app/resources/howto.html.template

```diff
@@ -0,0 +1,130 @@
+{% extends "static-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+<h1>Person Finder User’s Guide</h1>
+
+Person Finder can be used by anyone from a PC or mobile phone.
+This document shows how to provide and search for safety information in emergency situations.
+<div class="section">
+
+<h2>Provide your own safety information</h2>
+
+<p>
+1. Getting started.
+
+<p>
+Go to Google Person Finder homepage by entering the URL(<a href="http://g.co/pf">http://g.co/pf</a>)
+ in the browser and choose the disaster name relevant to you in the left side bar.
+<a href="https://google.org/personfinder/demo">Here</a> is an environment for demonstration so you can try the operation at any time.
+<p>
+<img src="png/en/home.png" width="300">
+<p>
+Click [I have information about someone] link after choosing the disaster name.<p>
+<img src="png/en/have_info.png" width="200">
+<p>
+2. Enter your name
```

Comment:
Can you be consistent whether to end the sentence with a period or not? Currently "1. Getting started." has a period while this line doesn't, etc.